### PR TITLE
Adds ability to scroll to bottom in Feed finishLoading

### LINKED ISSUE #85: Scroll to bottom by default
When reading through an issue, I think it would make sense to scroll the user to the last comment, rather than the first one.

Cheers!

### PR DESCRIPTION
**Pull request by flovilmart**
Closes #85

The issue controller actually doesn't support paging (for the moment, AFAICT), so scrolling to the last element will probably always be more comfortable / useful than staying at the top.

### CODE REVIEW

**Comment by flovilmart** [comment]
@rnystrom @BasThomas ACAICT even on long threads this seems to be OK.

File: Classes/Utility/UICollectionView+Utils.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return IndexPath(item: lastItem, section: lastSection)
+    }
+
+    func scrollToLastItem(at position: UICollectionViewScrollPosition = .top, animated: Bool) {
+        guard let indexPath = indexPathOfLastItem() else {
+            // empty collectionView
+            return
+        }
+        scrollToItem(at: indexPath, at: position, animated: false)
+    }
```
Comment (rnystrom) [inline Classes/Utility/UICollectionView+Utils.swift]: Quick question, why not just change the content offset based on the size+inset?

https://github.com/rnystrom/Freetime/blob/master/Classes/Issues/IssuesViewController.swift#L212-L214

Comment (flovilmart) [inline Classes/Utility/UICollectionView+Utils.swift]: That could work too :) tried with scrollToRectVisble but it didn’t work as expected

File: Classes/Utility/UICollectionView+Utils.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import UIKit
+
+extension UICollectionView {
+    func scrollToBottom(animated: Bool) {
+        var offset = contentOffset
+        offset.y = contentSize.height + contentInset.bottom - bounds.height
+        setContentOffset(offset, animated: animated)
+    }
+}
```
Comment (rnystrom) [inline Classes/Utility/UICollectionView+Utils.swift]: Hahaha, funny, I added literally the exact same extension on my own today.

**Comment by rnystrom** [comment]
The `IssuesViewController` class changed a lot upstream, might want to rebase and try this again. I can just take these changes and apply them to the branch manually too instead of trying to fix it up. Lmk.

**Comment by flovilmart** [comment]
Just cherry picked the changes. lmk

**Comment by rnystrom** [comment]
Took this and fixed merges, also executes after the refresh control finishes so the animations don't collide. Thanks @flovilmart!

**Pull request closed without merging**